app/src/main/java/org/mozilla/fenix/FenixApplication.kt

```diff
@@ -428,8 +428,6 @@ open class FenixApplication : LocaleAwareApplication(), Provider {
                 onNimbusStartupAndUpdate()
             }
         })
-
-        onNimbusStartupAndUpdate()
```

Comment:
Please could you file an issue to rename this (because it's no longer being called at startup).